app/helpers/hearing_helper.rb

```diff
@@ -8,4 +8,15 @@ def paginator(...)
   def earliest_day_for(hearing)
     hearing&.hearing_days&.map(&:to_datetime)&.sort&.first
   end
+
+  def display_crlf_and_sanitize(event_note)
+    note_with_converted_crlf = convert_crlf_to_html(event_note)
+    sanitize(note_with_converted_crlf, tags: %w[br p])
```

Comment:
do you need to whitelist `span` tag too? as you are using this instead of default `p` tag at line `20` below

Reply:
according to the definition of [santize here](https://apidock.com/rails/ActionView/Helpers/SanitizeHelper/sanitize)

Reply:
`simple_format` wraps `<p>` tags around the string by default, which would add extra spacing unnecessarily since it is whitelisted (to display `\r`) so I had to set it to something else that's not whitelisted to prevent it . I could also whitelist the `<span>` for it to render but since it makes no difference to the view, I chose not to clutter the view with unnecessary html

Reply:
Both `\n\n` or `\r\n\r\n` are considered a paragraph and are wrapped in `p` tags. If `p` is not whitelisted, then those will be sanitized out. Apologies I'm not doing a good job of explaining. 

Just for ref:
```
my_text = "Here is some basic text...\n...with a line break."

simple_format(my_text)
# => "<p>Here is some basic text...\n<br />...with a line break.</p>"

simple_format(my_text, {}, wrapper_tag: "div")
# => "<div>Here is some basic text...\n<br />...with a line break.</div>"

```
[simple_format documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html)